### DISCUSSION TITLE
Zeek issues

### DISCUSSION THREAD

**Discussion by dbui1910** [Security-Onion-Solutions/securityonion#5002] [Unsupported Versions] [answered]
Pining Zeek workers to specific CPUs but when getting into the minions folder there is no $SENSORNAME_$ROLE.sls file to edit .
The only file that exists in there is so-eval_eval.sls
Please help and thank you!

**Comment by TOoSmOotH** [comment, marked as answer]
The $ denotes a variable so $SENSORNAME would be so-eval and $ROLE is eval. So what you are looking for is in so-eval_eval.sls.
